vm-setup/roles/packages_installation/defaults/main.yml

```diff
@@ -47,19 +41,12 @@ packages:
          - OVMF
          - patch
          - psmisc
-         - python-pip
-         - python-requests
-         - python-setuptools
          - vim-enhanced
          - wget
          - ansible
```

Comment:
Do we need it here? Isn't it already installed by `centos_install_requirements.sh`, no?

Reply:
It could be removed but that's a different case from what I've tried to fix in this PR - I've removed things which we don't need anymore - in the case of ansible we definitely do need it, but due to the chicken/egg problem it's already installed.

For consistency I'd personally rather just leave all the packages we need in here - if the package is already installed this will do nothing but it helps see at a glance what the system dependencies are.

If there are strong opinions to the contrary we can fix it in a follow-up :)